README.md

```diff
@@ -3,20 +3,80 @@
 Read, collaborate and talk about scientific papers.
 
 ## Getting started
-1. Install Postgres
-2. Install `pandoc` to support references extraction from here: https://pandoc.org/installing.html
-3. Install `pdftotext` to support acronyms extraction: `sudo apt-get install poppler-utils`
-4. Create your Python 3.7 virtual env
-5. Create `.env` file based on the `.env_example` and fill out the values
-6. Run `flask fetch-papers` to grab some papers (you can stop the function after fetching ~200 papers) 
-7. Run `flask run`
-8. See `SciHive.postman_collection.json` for some examples of queries
 
+### 1) Install Postgres
+
+```
+podman run --name postgresql -p 5432:5432 -d postgres:latest
+```
+
+### Install pgadmin (optional)
+
+Change the default settings.
+
+```
+podman run --name pgadmin -p 5050:80 -e "PGADMIN_DEFAULT_EMAIL=pgadmin4@pgadmin.org" -e "PGADMIN_DEFAULT_PASSWORD=admin" -d dpage/pgadmin4
+```
+
```

Comment:
I don't think that this is necessary here

Reply:
It is optional. At least, people will know it is an option they can use and explore. It is a simple podman command to get started. Ideally, you would have a Dockerfile with a docker-compose file in this project. The setup and configuration time would have been saved for anyone who wants to contribute to this project.

For your information, Podman 2.0 was released less than one month ago.

Reply:
I agree in regards to Docker :)
https://github.com/ranihorev/scihive-backend/pull/9